src/internal/middleware/auth/interceptor.go

```diff
@@ -77,6 +77,9 @@ var authHandlers = map[string]authHandler{
 	"/enterprise_v2.API/GetActivationCode": authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_GET_CODE)),
 	"/enterprise_v2.API/Deactivate":        authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_DEACTIVATE)),
 	"/enterprise_v2.API/Heartbeat":         authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_HEARTBEAT)),
+	"/enterprise_v2.API/Pause":             authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_PAUSE)),
+	"/enterprise_v2.API/Unpause":           authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_PAUSE)),
+	"/enterprise_v2.API/PauseStatus":       authDisabledOr(clusterPermissions(auth.Permission_CLUSTER_ENTERPRISE_PAUSE)),
```

Comment:
The UX for a non-Admin learning that their cluster is paused seems like it may be lacking. It might make sense to have the PauseStatus API be available for users other than ClusterAdmin. I think it would be worth clarifying this with product

Reply:
If this is a status API that merely allows the user to find out if the cluster is paused, that's a great point. Any user with cluster access should be able to know the cluster state.  I believe we may have a little feature debt in pachctl that preceeds any API call with a "Hey, is the cluster paused?" call so that we can give them an informative error message rather than just hanging.  How can we best indicate that debt and get the work in the queue?

Reply:
I think that can be handled with a dedicated PPS & PFS implementation which returns an appropriate error code for every API call. Looking at https://pkg.go.dev/google.golang.org/grpc/codes, I think a decent argument can be made for either `FailedPrecondition` or `Unavailable`. I would like to do this as a follow-on story though.

Reply:
If the cluster is paused the other API servers (PFS & PPS) won't be active.  So GRPC will probably say the method doesn't exist.  We can change what is printed to the user in pachctl, but I don't think we should change the fact that those services aren't even running.

Reply:
We could add logic to pachctl that would run the PauseStatus API under the condition that it gets "service doesn't exist" error. Agreed those changes should be handled in a separate PR